Systematics/python/flashggDiPhotonSystematics_cfi.py

```diff
@@ -730,10 +733,36 @@
          BinList = emptyBins,
          CorrectionFile = scalesAndSmearingsPrefix,
          ApplyCentralValue = cms.bool(False),
+         UncertaintyBitMask = cms.string("011"),#cms.string("110"),
          ExaggerateShiftUp = cms.bool(False),
          Debug = cms.untracked.bool(False)
          )
 
+MCScaleGain6EB_EGM = cms.PSet( PhotonMethodName = cms.string("FlashggPhotonScaleEGMTool"),
+         MethodName = cms.string("FlashggDiPhotonFromPhoton"),
+         Label = cms.string("MCScaleGain6EB"),
+         NSigmas = cms.vint32(-1,1),
+         OverallRange = cms.string("hasSwitchToGain6&&abs(superCluster.eta)<1.5"),
+         BinList = emptyBins,
+         CorrectionFile = scalesAndSmearingsPrefix,
+         ApplyCentralValue = cms.bool(False),
+         ExaggerateShiftUp = cms.bool(False),
+         UncertaintyBitMask = cms.string("100"),#cms.string("001"), # this should be a bit mask, don't know how to make it in python now
```

Comment:
Bit masks in python are as simple as in C++
UncertaintyBitMask = cms.uint32(0x4),

Bitwise operators too
https://wiki.python.org/moin/BitwiseOperators

UncertaintyBitMask = cms.uint32(0x3), 
or 
UncertaintyBitMask = cms.uint32(0x2 | 0x1),